apps/etterlatte-pdltjenester/src/main/kotlin/person/PersonRoute.kt

```diff
@@ -10,24 +10,26 @@ import io.ktor.server.routing.post
 import io.ktor.server.routing.route
 import no.nav.etterlatte.libs.common.person.HentFolkeregisterIdentRequest
 import no.nav.etterlatte.libs.common.person.HentPersonRequest
+import no.nav.etterlatte.libs.ktor.SaksbehandlerProvider
 
-fun Route.personApi(service: PersonService) {
+fun Route.personApi(service: PersonService, saksbehandlerProvider: SaksbehandlerProvider) {
```

Comment:
Dette er eit triks for å kunne overstyre denne i testane.

Reply:
Nice det!

Reply:
Hvordan mocket vi dette andre steder? 🤔